cpp/src/ns/ns_evolution.cpp

```diff
@@ -471,6 +499,57 @@ double FreddiNeutronStarEvolution::R_Alfven() const {
 	return ns_str_->args_ns.R_Alfven(GM(), Mdot_in());
 }
 
+double FreddiNeutronStarEvolution::R_Alfven_basic() const {
+	return ns_str_->args_ns.R_Alfven_basic(GM(), Mdot_in());
+}//надо переписать это так же, как mu_magn
+
+
+
+double FreddiNeutronStarEvolution::R_Magn_bozzo18() const {
+	const double chi_oblique_rad = double(chioblique()) * M_PI / 180.;
+    const double alpha = 0.5; // как его взять честно??? его надо откуда-то вызвать
+	//eta (Bozzo2018) = 0.2, 
+	const double parametr_bozzo = 2. * m::pow<2>(0.2) / alpha; // на самом деле делить на alpha, как это сделать?
+	const double RA = R_Alfven_basic();
+    const double RCorrot = R_cor();
+	const double parametr_thetta = RCorrot  / RA;
+
+    //const vecd& H = Height();//почему так нельзя?
+    const double H_to_R_temp = h2rbozzo(); //временно
```

Comment:
`const vecd& H = Height();` is ok in terms of C++ but Freddi has non-physical vertical structure for inner part of the disc